amp.php

```diff
@@ -379,13 +379,12 @@ function amp_init() {
 	add_action( 'parse_query', 'amp_correct_query_when_is_front_page' );
 	add_action( 'admin_bar_menu', 'amp_add_admin_bar_view_link', 100 );
 	add_action( 'wp_loaded', 'amp_editor_core_blocks' );
+	add_action( 'amp_plugin_update', 'remove_amp_story_templates' );
```

Comment:
Story related templates will be removed after a plugin update; see https://github.com/ampproject/amp-wp/commit/d67421e0c73ecccd20f9b3db9980f6edf3157286.